.travis.yml

```diff
@@ -15,24 +15,22 @@ addons:
           key_url: "https://packages.microsoft.com/keys/microsoft.asc"
         packages:
             - powershell
-            - tree
 
 before_install:
-  - chmod ugo+x ./scripts/setup_repo.sh && ./scripts/setup_repo.sh
-  - tree /home/travis/build
-  - "echo \"work dir: $(pwd)\""
+  - chmod ugo+x ./scripts/setup_repo.sh
```

Comment:
we could avoid this line by making it executable in git:
`git update-index --chmod=+x ./scripts/setup_repo.sh`